documentation/4.0/content/samples/domains/model-in-image/_index.md

```diff
@@ -9,10 +9,19 @@ description: "Sample for supplying a WebLogic Deploy Tooling (WDT) model that th
 
 ### Introduction
 
-This sample demonstrates deploying a Model in Image [domain home source type]({{< relref "/managing-domains/choosing-a-model/_index.md" >}}). Unlike Domain in PV and Domain in Image, Model in Image eliminates the need to pre-create your WebLogic domain home prior to deploying your Domain YAML file. Instead, Model in Image uses a WebLogic Deploy Tooling (WDT) model to specify your WebLogic configuration.
+This sample demonstrates deploying a Model in Image
+[domain home source type]({{< relref "/managing-domains/choosing-a-model/_index.md" >}})
+ with [Auxiliary images]({{< relref "/managing-domains/model-in-image/auxiliary-images.md" >}}).
+Unlike Domain in PV and Domain in Image, Model in Image eliminates the need to pre-create
+your WebLogic domain home prior to deploying your Domain YAML file.
+Instead, Model in Image uses
+WebLogic Deploy Tooling (WDT) model to specify your WebLogic configuration.
 
 WDT models are a convenient and simple alternative to WebLogic Scripting Tool (WLST) configuration scripts and templates. They compactly define a WebLogic domain using YAML files and support including application archives in a ZIP file. The WDT model format is described in the open source, [WebLogic Deploy Tooling](https://oracle.github.io/weblogic-deploy-tooling/) GitHub project, and the required directory structure for a WDT archive is specifically discussed [here](https://oracle.github.io/weblogic-deploy-tooling/concepts/archive/).
 
+The Model in Image auxiliary image option furthermore allows you to supply your WDT artifacts
```

Comment:
-> Furthermore, the Model in Image auxiliary image option lets you

Reply:
done